src/components/HintIcon/HintIcon.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import { omit } from 'ramda';
+import styled from 'styled-components';
+
+import HintIcon from './HintIcon';
+import { HintIconProps } from './HintIcon.types';
+import { PortalPlacements } from '../../hooks/useCalculatePortalPlacements.enums';
+import { Paragraph, Text } from '../typographyLegacy';
+import { Button } from '../Button';
+import { Inline, Padbox, Stack } from '../layout';
+import { generateControl } from '../../utils/tests/storybook';
+import { SpaceSizes } from '../../theme';
+import { getColor } from '../../utils';
+
+export default {
+  title: 'components/HintIcon',
+  component: HintIcon,
+  argTypes: {
+    popup: {
+      control: { disable: true },
+    },
+
+    placement: {
+      ...generateControl(
+        'select',
+        omit(['bottomLeft', 'bottomRight'], PortalPlacements),
+      ),
+    },
+  },
+  decorators: [
+    (storyFn) => (
+      <Padbox paddingSize={SpaceSizes.xxl}>
+        <Inline justify="center">{storyFn()}</Inline>
+      </Padbox>
+    ),
+  ],
+} as Meta;
+
+const Popup = (
+  <Stack align="center" gap="md" justify="center">
+    <Paragraph size="md">This is the tooltip content</Paragraph>
+    <Button variant="text">Here is some link</Button>
+  </Stack>
+);
+
+const ColorBackgroundWrapper = styled(Padbox)`
+  background-color: ${getColor('primary.50')};
+`;
+
+export const Default: Story<HintIconProps> = ({ ...args }) => (
+  <Inline>
+    <Padbox height="120" paddingSize="lg" width="240">
+      <HintIcon placement="top" {...args} />
+    </Padbox>
+    <ColorBackgroundWrapper height="120" paddingSize="lg" width="240">
+      <HintIcon placement="top" {...args} />
+    </ColorBackgroundWrapper>
```

Comment:
What's the purpose of the coloured background here?

Reply:
to make the white background visible